src/main/frontend/components/whiteboard.cljs

```diff
@@ -253,7 +259,12 @@
                         (fn [open? refs-count] [:span.whiteboard-page-refs-count-label
                                                 (if (> refs-count 1) "References" "Reference")
                                                 (ui/icon (if open? "references-hide" "references-show")
-                                                         {:extension? true})]))]]
+                                                         {:extension? true})]))]
+      (dropdown-menu {:label "(DEV) Show blocks in outliner"
+                      :classname "hidden"
```

Comment:
The font color is not readable if the dark theme is enable.

Suggestion: We could use `config/dev?` to completely remove this from the dom tree on prod. Maybe always show it on dev mode? In the future, it might be useful to introduce a `debug?` mode for Logseq (not just whiteboards), to enable visual helpers like this one, and like the undo/redo stack.

We could also also try to take this flag into account in this case, to streamline the development process
https://github.com/logseq/logseq/blob/3940db310719733c5d203c1cee5ee4c682f5193b/tldraw/apps/tldraw-logseq/src/components/AppUI.tsx#L7

Reply:
I will go with the shift+clicking solution later. For this PR I will revert these changes.